storybook/qmlTests/tests/tst_OwnerTokenWelcomeView.qml

```diff
@@ -0,0 +1,86 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+import QtTest 1.15
+
+import StatusQ 0.1
+
+import SortFilterProxyModel 0.2
+import AppLayouts.Communities.views 1.0
+
+import Models 1.0
+
+Item {
+
+    id: root
+    width: 600
+    height: controlUnderTest ? controlUnderTest.implicitHeight : 800
+
+    Component {
+        id: componentUnderTest
+
+        OwnerTokenWelcomeView {
+
+            anchors.fill: parent
+
+            communityLogo: ModelsData.icons.superRare
+            communityColor: "pink"
+            communityName: "SuperRare"
+        }
+    }
+
+    property OwnerTokenWelcomeView name: OwnerTokenWelcomeView
+    property var controlUnderTest
+
+    SignalSpy {
+        id: nextClickSpy
+        target: controlUnderTest
+        signalName: "nextClicked"
+    }
+
+    TestCase {
+        name: "OwnerTokenWelcomeView"
+        when: windowShown
+
+        function init() {
+            nextClickSpy.clear()
+            controlUnderTest = createTemporaryObject(componentUnderTest, root)
+        }
+
+        function htmlToPlainText(html) {
+            return html.replace(/<[^>]+>/g, "")
+        }
+
+        function test_ui_elements() {
+            waitForRendering(controlUnderTest)
+
+            const introText = findChild(controlUnderTest, "introPanelText")
+            const ownerInfoPanel = findChild(controlUnderTest,
+                                             "infoPanel_owner")
+            const masterInfoPanel = findChild(controlUnderTest,
+                                              "infoPanel_master")
+
+            verify(introText !== null, "Intro text is missing")
+            verify(ownerInfoPanel !== null, "Owner info panel is missing")
+            verify(masterInfoPanel !== null, "Master infor panel is missing")
```

Comment:
there seems to be a type also `infor`, maybe you want is `compare` ?

Reply:
ah right! sorey its a typo, will fix!